src/app/features/admin-home/components/admin-registration-version/admin-registration-version.component.ts

```diff
@@ -1,8 +1,31 @@
-import { Component } from "@angular/core";
+import { Component, OnInit } from "@angular/core";
+import { Store, select } from "@ngrx/store";
+import { Observable } from "rxjs";
+import { AdminHomeService } from "../../services/admin-home-service";
+import * as AdminHomeActions from "../../store/actions";
+import { AdminState } from "../../store/reducers";
+import { probesInfo } from "../../../../shared/types/probesInfo.interface";
 
 @Component({
   selector: "admin-registration-version",
   templateUrl: "./admin-registration-version.component.html",
   styleUrls: ["./admin-registration-version.component.scss"],
 })
-export class AdminRegistrationVersionComponent {}
+export class AdminRegistrationVersionComponent implements OnInit {
+  probesInfo$: Observable<probesInfo>;
+  error$: Observable<any>;
+
+  constructor(
+    private store: Store<{ admin: AdminState }>,
+    private adminHomeService: AdminHomeService
+  ) {
+    this.probesInfo$ = this.store.pipe(
+      select((state) => state.admin.probesInfo)
```

Comment:
use ?. to check for null/undefined property value

Reply:
done